packages/keymaps/src/browser/keybindings-widget.tsx

```diff
@@ -440,19 +440,21 @@ export class KeybindingWidget extends ReactWidget {
         for (let i = 0; i < commands.length; i++) {
             // Obtain the keybinding for the given command.
             const keybindings = this.keybindingRegistry.getKeybindingsForCommand(commands[i].id);
-            const item: KeybindingItem = {
-                id: commands[i].id,
-                // Get the command label if available, else use the keybinding id.
-                command: commands[i].label || commands[i].id,
-                keybinding: (keybindings && keybindings[0]) ? keybindings[0].keybinding : '',
-                context: (keybindings && keybindings[0])
-                    ? keybindings[0].context
-                        ? keybindings[0].context : keybindings[0].when
-                    : '',
-                source: (keybindings && keybindings[0] && typeof keybindings[0].scope !== 'undefined')
-                    ? KeybindingScope[keybindings[0].scope!].toLocaleLowerCase() : '',
-            };
-            items.push(item);
+            for (const keybindingCmd of keybindings) {
+                const item: KeybindingItem = {
+                    id: commands[i].id,
+                    // Get the command label if available, else use the keybinding id.
+                    command: commands[i].label || commands[i].id,
+                    keybinding: (keybindings && keybindingCmd) ? keybindingCmd.keybinding : '',
```

Comment:
`keybindings && keybindingCmd` is not it always true? Since we iteratere `keybindings` it cannot be undefined, and since we get an element `keybindingCmd` it cannot be `undefined` either.

I think the `for body` can be simplified by removing redundant checks.